frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -12,7 +12,7 @@ import {
 import { ProjectSectionID } from './types';
 
 type DetailsSectionProps = {
-  id: ProjectSectionID;
```

Comment:
My question bellow is related to this, I'm not sure why we need to convert the id to a general string, the hole power of enums is that you limit the scope of the ids, in this case the ones defined in the enum, shouldn't we stick to this type here?

Reply:
Same change was made in this PR
https://github.com/opendatahub-io/odh-dashboard/pull/983#discussion_r1126748763

I would do the same thing here @jenny-s51 that you did in the comment linked

Reply:
Rebased - this file should be up to date now